app/components/work_packages/types/pattern_autocompleter.sass

```diff
@@ -0,0 +1,5 @@
+.pattern-autocompleter
+  .selected
+    color: var(--list-item-hover--color)
```

Comment:
Looking at the ActionList component, they use `--fgColor-default` for hovered elements. As we try to mimic that style, I'd suggest that we use the same here.